src/main/java/stsjorbsmod/cards/cull/Discordance.java

```diff
@@ -0,0 +1,68 @@
+package stsjorbsmod.cards.cull;
+
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.common.DamageAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import stsjorbsmod.JorbsMod;
+import stsjorbsmod.cards.CustomJorbsModCard;
+import stsjorbsmod.characters.Cull;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+
+public class Discordance extends CustomJorbsModCard {
+    public static final String ID = JorbsMod.makeID(Discordance.class);
+
+    private static final CardRarity RARITY = CardRarity.UNCOMMON;
+    private static final CardTarget TARGET = CardTarget.ENEMY;
+    private static final CardType TYPE = CardType.ATTACK;
+    public static final CardColor COLOR = Cull.Enums.CULL_CARD_COLOR;
+
+    private static final int COST = 1;
+
+
+
+    public Discordance() {
+        super(ID, COST, TYPE, COLOR, RARITY, TARGET);
+        this.baseDamage = 0;
+        this.baseMagicNumber = magicNumber = 5;
+    }
+
+    @Override
+    public int calculateBonusBaseDamage() {
+        Iterator var2 = AbstractDungeon.player.hand.group.iterator();
```

Comment:
It isn't clear to me from the spreadsheet's text whether this should count itself being an attack for the purposes of "play next card twice" effects (eg, Shrapnel Bloom) and/or "play card from a non-hand position" (eg, Channel); as-is, it will count itself in simple "play from hand" cases, but won't count itself in either of those 2 special cases. I asked in #cull-contributing on Discord what the intended behavior should be; we might need to adjust this to explicitly add `this.type` to the set of types considered depending on the answer.

Reply:
Per discord discussion, let's go with consistency with how Second Wind and Fiend Fire don't consider themselves as being "in-hand" while they're being played, even in the basic case; ie, explicitly skip `this` from consideration as part of the `seen` list

Reply:
Updated behavior to skip self